gems/pending/metadata/linux/LinuxSystemd.rb

```diff
@@ -41,6 +41,14 @@ def files(unit_extension)
       end
     end
 
+    def my_dirs
+      dirs = []
+      (SYSTEM_DIRS + USER_DIRS).each do |dir|
+        dirs << dir if @fs.fileExists?(dir)
+      end
+      dirs
```

Comment:
In addition, can the constants be added together ahead of time into a new constant?